src/components/Box/Box.tsx

```diff
@@ -10,11 +10,12 @@ import "./index.scss";
 export interface BoxProps {
   children: React.ReactNode;
   title: string;
+  dataQA?: string | null;
```

Comment:
Please rename `dataQA` prop to `dataQa` to be consistent with prop in [other components](https://github.com/AccelByte/justice-ui-library/blob/master/src/components/Badge/Badge.tsx#L21)

Reply:
changed, thankyou